flake.nix

```diff
@@ -0,0 +1,50 @@
+{
+  epoch = 201909;
+
+  description = "A collection of packages for the Nix package manager";
+
+  outputs = { self }:
+    let
+      pkgs = import ./. { system = "x86_64-linux"; };
```

Comment:
does this mean we currently can't use the nixpkgs flake from any non-x86 system?

Reply:
Not from the `nix` command line. You can call it from other flakes with a non-x86_64-linux system argument though.

Reply:
Can't we use builtins.currentSystem?

Reply:
Nope, not in pure evaluation mode.

```
$ nix eval "(builtins.currentSystem)"
error: attribute 'currentSystem' missing, at (string):1:2
$ nix eval --impure "(builtins.currentSystem)"
"x86_64-linux"
```